microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -5,15 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 08/18/2023
+ms.date: 12/13/2023
 ---
 # Microsoft Edge Add-ons store developer policies
 
-
-<!-- ====================================================================== -->
-## Introduction and Objective of this document
-
-Thank you for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
+Thank you<!-- todo: go easy on "thank you", per corp-level style guide --> for your interest in developing extensions for the Microsoft Edge Add-ons store.  The Microsoft Edge Add-ons store developer policies (Add-ons store developer policies) apply to your extensions, including your submission of extensions through [Partner Center](https://partner.microsoft.com/dashboard/microsoftedge/public/login?ref=dd) and the provision of such extensions through the Microsoft Edge Add-ons.
 
 The _Microsoft Edge Add-ons store_ is also called the _Microsoft Edge Add-ons website_.  The _Microsoft Edge Add-ons store developer policies_ are also called the _Microsoft Edge Add-ons website developer policies_.
```

Comment:
```suggestion
The _Microsoft Edge Add-ons store_ is also called the _Microsoft Edge Add-ons website_.  The _Microsoft Edge Add-ons store developer policies_ are also called the _Add-ons store developer policies_ or the _Microsoft Edge Add-ons website developer policies_.
```

Reply:
updated link text from website to store, then deleted:
The _Microsoft Edge Add-ons store developer policies_ are also called the _Add-ons store developer policies_ or the _Microsoft Edge Add-ons website developer policies_.